app/hooks/contextHooks.ts

```diff
@@ -1,10 +1,10 @@
 import { useOutletContext } from '@remix-run/react';
 import { AppContext } from '~/typer/context';
-import { ESanitySteg } from '~/typer/sanity/sanity';
+import { ESanitySteg, Tekstinnhold } from '~/typer/sanity/sanity';
 
 export function useTekster(steg: ESanitySteg) {
   const { sanityTekster } = useOutletContext<AppContext>();
-  return sanityTekster[steg];
+  return sanityTekster[steg] as Tekstinnhold;
```

Comment:
Forslag: Beholde denne uten å returnere et spesifikt steg, men heller alle tekster. Da kan man hente ut teksten for steget som det nå gjøres i `index` uten å måtte bruke `as` og vi slipper å bruke `useOutletContext<AppContext>()` på alle sider. Usikker på om dette er bedre, men syns selv at `useTekster()` ser litt cleanere ut og det blir likt som hvordan vi bruker språk

Reply:
Da trenger ikke denne metoden å ta inn steg

Reply:
Enig i at det er fint å gjøre det på samme måte som med useSpråk. Ble en blanding av begge endringene nå her [8c176ef](https://github.com/bekk/nav-familie-endringsmelding/pull/31/commits/8c176eff96cf1fbf2629b4bd1c9596d0b70fd21b)